docs/scripts/degen.rst

```diff
@@ -0,0 +1,67 @@
+degen
+=====
+Find genes where a sequence has degenerate bases.
+
+How-to
+=====
+Usage:
```

Comment:
Maybe just the options? They look pretty in RTD: http://bio-pieces.readthedocs.org/en/dev/scripts/degen.html